shared/fixes/bash/rpm_verify_permissions.sh

```diff
@@ -22,7 +22,7 @@ do
 done
 
 # Remove duplicate mention of same RPM in $SETPERMS_RPM_LIST (if any)
-SETPERMS_RPM_LIST=( $(echo "${SETPERMS_RPM_LIST[@]}" | sort -n | uniq) )
+SETPERMS_RPM_LIST=( $(echo "${SETPERMS_RPM_LIST[@]}" | tr ' ' '\n' | sort -u | tr '\n' ' ') )
```

Comment:
@iokomin do you mind commenting why this fix is needed and what it fixed? Might also want to change the comment above.

Reply:
@redhatrises, this line fixes initial duplicates removal intended to work but doesn't in fact. 'echo' produces string with space separated array elements which 'sort -n' can't handle as it works with lines. Suggested fix replaces delimiter to new line one, sorts with leaving unique values and converts it back to array - as expected and stated in the above comment.

Reply:
Please find examples below..
original code
```
$declare -a SETPERMS_RPM_LIST=(element1 element2 element2); SETPERMS_RPM_LIST=( $(echo "${SETPERMS_RPM_LIST[@]}" | sort -n | uniq) ); echo ${SETPERMS_RPM_LIST[@]}
element1 element2 element2
```
fixed code
```
$declare -a SETPERMS_RPM_LIST=(element1 element2 element2); SETPERMS_RPM_LIST=( $(echo "${SETPERMS_RPM_LIST[@]}" | tr ' ' '\n' | sort -u | tr '\n' ' ') ); echo ${SETPERMS_RPM_LIST[@]}
element1 element2
```

Reply:
@iokomin thanks. Needed that for historical reasons.